src/Days.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace IconLanguageServices\Time;
+
+final class Days implements ConvertsToSeconds

```

Comment:
It only returns in seconds because that's our use case at the moment. If we determine that we need to get times back in other units, we can add those as necessary.